lms/static/discussion

```diff
@@ -0,0 +1 @@
+../djangoapps/discussion/static/discussion
```

Comment:
This is interesting, I wonder if there's any way to configure things so this isn't necessary. (One of the primary reasons edx-platform won't run on Windows is because of symlinks, and were starting to think about ways to eliminate them.)